tests/modules/asset_groups/resources/utils.py

```diff
@@ -450,6 +462,40 @@ def patch_asset_group_sighting(
     return response
 
 
+def patch_asset_group_sighting_as_sighting(
```

Comment:
Unnecessary duplication.
Would be better to  tweak patch_asset_group_sighting to have the sighting bit of the path configurable via the calling api that defaulted to "sighting".